docs/testcases/UI/Billing/762/762_5.md

```diff
@@ -0,0 +1,61 @@
+| 8 | Click into the ***User*** field and select non-admin from the prerequisites | | 
+| 9 | Specify follow values for quota <li> *Quota* - `<billing2>` <li> quota period -`per year` <li> *Threshold* - `70` <li> *Action* - `Disable new jobs` | | 
+| 10 | Click **Save** button | The following info appears in the ***Users*** section: <li> `<Non_admin_user>: <quota>$ per year`, where `<quota>` is value specified at step 5 <li> `70%: disable new jobs` - pair of threshold-action configured at step 5 |
+| 11 | Wait time equals to `<billing_quotas_period>` from the [762_1](762_1.md) case Preparations and refresh page | ***Red*** circles are shown near the quotas created at steps 6 and 10 | 
+| 12 | Logout and login as non-admin user from the prerequisites | |
+| 13 | Open the **Runs** page. Open **Completed runs** tab | |
+| 14 | Check run launched at step 4 from the Preparations | Run has *Stopped* status |
+| 15 | Click **Rerun** link for the run launched at step 4 from the Preparations | Error message `Launch of new compute instances is forbidden due to exceeded billing quota` appears | 
+| 16 | Open **Active runs** tab and check run launched at step 5 from the Preparations | Run has *Paused* status |
+| 17 | Click **Resume** link for run launched at step 5 from the Preparations | Error message `Launch of new compute instances is forbidden due to exceeded billing quota` appears |
+| 18 | Open the **Tools** page | |
+| 19 | Select test tool | |
+| 20 | Launch a selected tool with default settings | Error message `Launch of new compute instances is forbidden due to exceeded billing quota` appears |
+| 21 | Logout and login as admin user from the prerequisites | |
+| 22 | At the **Runs** page, click the run launched at step 9 of the Preparations | |
+| 23 | Wait until the **SSH** hyperlink appears and click it | |
+| 24 | Execute command: <br> `pipe run -di <test_tool> -u <non-admin_user>`, <br> where `<test_tool>` is the test tool name | Response `Error: Failed to fetch data from server. Server responded with message: Launch of new compute instances is forbidden due to exceeded billing quota.` is returned |
+| 25 | Open the **Settings** page | |
+| 26 | Click the **USER MANAGEMENT** tab | |
+| 27 | Click the **Users** tab | |
+| 28 | Find the non-admin user from the prerequisites | User has warning `Billing quotas exceeded` |
+| 29 | Hover over the warning | Tooltip that contains info <br> `Billing center <Billing_center>: compute quarterly expenses <...>$,quota <billing1>$. Actions:70% Disable new jobs` <br> `User <non_admin_user>: compute annual expenses <...>$,quota <billing2>$. Actions:70% Stop all jobs` |
+| 30 | Find the admin user from the prerequisites | User hasn't warning `Billing quotas exceeded` |
+
+**After:**
+- Remove quotas created at steps 6, 10
```

Comment:
Add rows:
- Terminate the run paused at step 7 of the case preparations
- Stop the run launched at step 9 of the case preparations

Reply:
Done